apps/predict-ui/src/components/ui/Pill.tsx

```diff
@@ -0,0 +1,47 @@
+import { Badge, Flex } from 'antd';
+import { CSSProperties, ReactNode } from 'react';
+
+import { COLOR } from '../../constants/theme';
+import { GLOBAL_COLORS } from '@agent-ui-monorepo/ui-theme';
+
+type PillProps = {
+  type?: 'primary' | 'danger';
+  size?: 'small' | 'large';
+  style?: CSSProperties;
+  children: ReactNode;
+};
+
+// TODO: add a ui-pill nxx lib and move this there
+export const Pill = ({ type, size = 'small', style, children }: PillProps) => {
+  return (
+    <Flex
+      gap={size === 'small' ? 4 : 8}
+      align="center"
+      style={{
+        display: 'inline-flex',
+        padding:
+          size === 'small' ? `2px 4px 2px ${type ? 16 : 8}px` : `6px 12px 6px ${type ? 12 : 6}px`,
+        borderRadius: 40,
+        marginLeft: -28,
```

Comment:
in almost all places except in AssetBadge we reset margin left to 0, maybe we should have it here instead, and apply needed margin in AssetBadge?